README.md

```diff
@@ -48,10 +48,12 @@ Specify `$WANDB_ENTITY`, `$WANDB_PROJECT`, `$WANDB_NAME` environment variables p
 
 # Launching
 
-### GPU requirements
-This code was developed and tested using a single A100 GPU with 80GB GPU RAM. It may successfully run on GPUs with 32 - 40GB for perplexity evaluation of up to `LLaMA-65B` and `Falcon-40B` models.
+### GPU and RAM requirements
+This code was developed and tested using a single A100 GPU with 80GB GPU RAM. It may successfully run on GPUs with 32GB+ VRAM for perplexity evaluation of up to `LLaMA-65B` and `Falcon-40B` models. 
+With `--offload activations` option, the model perplexity may be evaluated on machines with less VRAM: 24GB+ for Llama 65B and 6GB+ for Llama 7B.
 For `Language Model Evaluation Harness` evaluation one needs to have enough memory to load whole model
 on one or several devices + activation tensors. 
+This code also requires RAM amount sufficient to hold uncompressed model weights (e.g. ~130GB for Llama65B) and testing datasets.
```

Comment:
This should be after line 54

Reply:
The code requires RAM  sufficient to hold uncompressed model weights (e.g. ~130GB for Llama65B) and testing datasets.